src/server/api/controllers/channel-members.controller.js

```diff
@@ -40,16 +42,23 @@ const createChannelMember = async (body) => {
     successful: true,
   };
 };
-async function checkForCommoneChannels(arrOfUsers) {
-  // get all channels where the users are member off and the number of users is equal to arrOfUsers.length
-  const allChannelsForUsers = await knex('channel_members as a')
-    .count('a.fk_user_id', { as: 'numberOfUsers' })
-    .join('channel_members as b', 'b.fk_channel_id', 'a.fk_channel_id')
-    .select('a.id', 'a.fk_channel_id as channelId')
-    .whereIn('a.fk_user_id', arrOfUsers)
-    .groupBy('a.id')
-    .having('numberOfUsers', '=', arrOfUsers.length);
+const checkForCommoneChannels = async (arrOfUsers) => {
```

Comment:
I'd rather you didn't fix this code and instead approached Wael to ensure his code works. Then your entire PR would be smaller and more concise.
Even though your code is much shorter.